src/tsconfig.json

```diff
@@ -21,7 +21,7 @@
     // Prevent automatic inclusion of global variables defined in `@types/<name>` packages.
     // This prevents eg. Node globals from `@types/node` being included when writing
     // code for the browser.
-    "types": []
+    "types": ["mocha"]
```

Comment:
Needed for global functions such as `describe`, `it`, `beforeEach`, `afterEach`, etc., to be properly detected by typescript.